src/main/java/io/jenkins/plugins/orka/client/ConfigurationRequest.java

```diff
@@ -26,11 +26,17 @@
     @SuppressFBWarnings("URF_UNREAD_FIELD")
     private int cpuCount;
 
-    public ConfigurationRequest(String vmName, String image, String baseImage, String configTemplate, int cpuCount) {
+    @SerializedName("scheduler")
```

Comment:
No need for the annotation. It's intended to be used if we want the serialized field to have different name than the field identifier. In our case they are the same